.github/workflows/ci.yml

```diff
@@ -30,7 +46,19 @@ jobs:
         run: bin/readme
       - name: Run tests
         run: bin/test
+        continue-on-error: ${{ matrix.experimental }}
       - name: Verify gem RBIs are up-to-date
         run: bundle exec exe/tapioca gem --verify
+        if: ${{ matrix.gemfile == 'Gemfile' }}
       - name: Verify duplicates in shims
         run: bundle exec exe/tapioca check-shims
+
+  buildall:
+    if: ${{ always() && github.event.pull_request }}
```

Comment:
Out of curiosity: why do we need `always()` here? Would this not work the same way if it was `${{ github.event.pull_request }}` only?

Reply:
Without it, If the build step fails, this will not run, so we will not have the failing status.

Reply:
Ah, got it, thanks.